charts/sourcegraph-migrator/templates/migrator/sourcegraph-migrator.Job.yaml

```diff
@@ -0,0 +1,75 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  annotations:
+    description: Performs Sourcegraph database migrations.
+  labels:
+    {{- include "sourcegraph.labels" . | nindent 4 }}
+    {{- if .Values.migrator.labels }}
+      {{- toYaml .Values.migrator.labels | nindent 4 }}
+    {{- end }}
+    deploy: sourcegraph
+    app.kubernetes.io/component: migrator
+  name: {{ default "migrator" .Values.migrator.name }}-{{ randAlphaNum 5 | lower }}
```

Comment:
`{{ randAlphaNum 5 | lower }}` ensures user don't have to uninstall the chart every time.

Upon each upgrade, previous job will be delete by helm or user can clean them up manually.